pkg/flowkit/config/json/config.go

```diff
@@ -108,9 +121,25 @@ type Parser struct{}
 func NewParser() *Parser {
 	return &Parser{}
 }
+func emulatorIsDefault(emulator config.Emulator) bool {
+	if emulator.Name == config.DefaultEmulatorConfigName &&
+		emulator.ServiceAccount == config.DefaultEmulatorServiceAccountName &&
+		emulator.Port == config.DefaultEmulatorPort {
+		return true
+	}
+	return false
+}
```

Comment:
This shouldn't be defined in the `config.go` but should be part of `emulator.go` and already is, you have the `DefaultEmulator()` function you can use to compare.